### ISSUE TITLE
http://www.delfi.lt/ - leftovers

### ISSUE THREAD

**Issue by evgeniy-ADG** [AdguardTeam/AdguardFilters#6546]
[//]: # (***You can delete or ignore strings starting with "[//]:" They will not be visible either way.)

***Description***:
* **Current behaviour**: http://www.delfi.lt/news/daily/law/jauno-vyro-klausimai-sukele-tikra-samysi-policijoje-ir-teisme-paaiskejo-neitiketina-klaida.d?id=75610393

[//]: # (Substitute this line with a description of the problem)

[//]: # (Replace %screenshot_url% below with a link to the screenshot of the problem. Also, you can paste image from clipboard instead. It will be automatically loaded.)
<details><summary>Screenshot:</summary>

![image](https://user-images.githubusercontent.com/15856304/29919448-040e5c5e-8e52-11e7-9f79-fb39c330be29.png)

</details><br/>

***System configuration***

**Filters:**

Subscriptions: FilterId:2 Name: English filter Url: 

- https://filters.adtidy.org/extension/chromium/filters/2.txt
- FilterId:3 Name: Spyware filter Url: https://filters.adtidy.org/extension/chromium/filters/3.txt
- FilterId:4 Name: Social media filter Url: https://filters.adtidy.org/extension/chromium/filters/4.txt
- FilterId:10 Name: Filter unblocking useful ads Url: https://filters.adtidy.org/extension/chromium/filters/10.txt
- FilterId:1 Name: Russian filter Url: https://filters.adtidy.org/extension/chromium/filters/1.txt
- FilterId:6 Name: German filter Url: https://filters.adtidy.org/extension/chromium/filters/6.txt
- FilterId:9 Name: Spanish/Portuguese filter Url: https://filters.adtidy.org/extension/chromium/filters/9.txt
- FilterId:105 Name: EasyList Czech and Slovak Url: https://raw.github.com/tomasko126/easylistczechandslovak/master/filters.txt
- FilterId:110 Name: EasyList Lithuania Url: http://margevicius.lt/easylistlithuania.txt
- FilterId:121 Name: void.gr Url: http://www.void.gr/kargig/void-gr-filters.txt

[//]: # (Substitute this line with the list of your active filters, separated by commas)

[//]: # (Please enter the correct values for your case to the table below)

Information                         | Value
---                                 | ---
Operating system:                   | Mac \ Windows
Browser:                            | Chrome
AdGuard version:                    | Latest versions
AdGuard DNS:                        | None 
Helpdesk ID (if exists):            | 1591247

[//]: # (This template is meant for missed ad/false positive reports, for other type of reports edit it accordingly)
[//]: # (If this is a crash report, include the crashlog with https://gist.github.com/)

**Comment by Alex-302** [comment]
Not reproduced.
Added
```
delfi.lt##.right img[width="300"][height="600"]
delfi.lt##iframe[src^="https://www.delfi.lt/apps/banners/"]
```
@AdamWr please check it on your end.

**Comment by AdamWr** [comment]
I was able to reproduce.
Added:
`delfi.lt##body > div[style^="position:fixed;"][style*="z-index:9999999"]:not([class]):not([id])`
